caf-kubernetes-client/src/main/java/com/github/cafapi/kubernetes/client/KubernetesClientFactory.java

```diff
@@ -13,7 +13,6 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
```

Comment:
Did you mean to remove these lines?

Reply:
Looks like added by the license plugin itself.